recipes/vc-feature/meta.yaml

```diff
@@ -0,0 +1,23 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: vc-feature
+  version: {{ version }}
+
+build:
+  number: 0
+  track_features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
```

Comment:
I think we still need it initially because the features are selecting on Python, but that is probably a good idea to add to the `run` section. Do you have thoughts on this, @patricksnape?